### PR TITLE
Fix spelling

### PR DESCRIPTION
**Pull request by jbampton**
Thanks for your interest in plotly.js!

### Translations:

- Please @ mention a few other speakers of this language who can help review your translations.
- If you've omitted any keys from [dist/translation_keys.txt](https://github.com/plotly/plotly.js/blob/master/dist/translation-keys.txt) - which means they will fall back on the US English text - just make a short comment about why in the PR description: the English text works fine in your language, or you would like someone else to help translating those, or whatever the reason.
- You should only update files in `lib/locales/`, not those in `dist/`

### Features, Bug fixes, and others:

Before opening a pull request, developer should:

- `git rebase` their local branch off the latest `master`,
- make sure to **not** `git add` the `dist/` folder (the `dist/` is updated only on version bumps),
- make sure to commit changes to the `package-lock.json` file (if any new dependency required),
- write an overview of what the PR attempts to do,
- select the _Allow edits from maintainers_ option (see this [article](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) for more details).

Note that it is forbidden to force push (i.e. `git push -f`) to remote branches associated with opened pull requests. Force pushes make it hard for maintainers to keep track of updates. Therefore, if required, please `git merge master` into your PR branch instead of `git rebase master`.

### CODE REVIEW

File: test/jasmine/tests/mock_test.js [hunk 1]
```diff
@@ -1815,8 +1815,8 @@ figs['mapbox_choropleth0'] = require('@mocks/mapbox_choropleth0');
 figs['mapbox_choropleth0-legend'] = require('@mocks/mapbox_choropleth0-legend');
 figs['mapbox_connectgaps'] = require('@mocks/mapbox_connectgaps');
 figs['mapbox_custom-style'] = require('@mocks/mapbox_custom-style');
-figs['mapbox_density-mulitple'] = require('@mocks/mapbox_density-mulitple');
-figs['mapbox_density-mulitple_legend'] = require('@mocks/mapbox_density-mulitple_legend');
+figs['mapbox_density-multiple'] = require('@mocks/mapbox_density-multiple');
+figs['mapbox_density-multiple_legend'] = require('@mocks/mapbox_density-multiple_legend');
```
Comment (archmoj) [inline test/jasmine/tests/mock_test.js:1819]: Could you please rename the mock and baseline files at following addresses?

test/image/mocks/mapbox_density-mulitple.json  
test/image/mocks/mapbox_density-mulitple_legend.json
test/image/baselines/mapbox_density-mulitple_legend.png
test/image/baselines/mapbox_density-mulitple.png

File: test/jasmine/tests/range_slider_test.js [hunk 2]
```diff
@@ -266,7 +266,7 @@ describe('Visible rangesliders', function() {
                 rangeslider: {visible: true}
             },
             width: 800,
-            hieght: 500
+            height: 500
```
Comment (archmoj) [inline test/jasmine/tests/range_slider_test.js:269]: Good catch.
Let's remove this line, so that the default `height` is applied as before.

**Review by archmoj — COMMENTED** [review]
Looking good to me.
I am interested to know how you catch all these?
Please provide this info in the PR description instead of the template.
Thank you!

**Comment by archmoj** [comment]
:dancer:
Merging...
I'll edit the requested changes in the next PR.

**Comment by jbampton** [comment]
Hey @archmoj  I can do a PR now and fix them if you want ?

**Comment by archmoj** [comment]
> Hey @archmoj I can do a PR now and fix them if you want ?

I am about to submit a PR. So no worries there.
But please just edit the PR description here and if you let us know how you catch these? https://github.com/plotly/plotly.js/pull/5349#pullrequestreview-554840552
I am curious to know if we could possibly add a test to avoid these mistakes.

**Pull request merged**